bridge_adaptivity/module/templates/module/activity_modal_form.html

```diff
@@ -27,15 +27,15 @@
         <button
                 class="btn btn-default activity-show-advanced-options"
                 data-toggle=".advanced_fields"
-                onclick="$($(this).data('toggle')).toggle('slow'); event.preventDefault();">
```

Comment:
I would update `onclick` button's parameter instead. Why it won't work, what do you think?

Reply:
I  rebased js code into `onclick` parameter.

Reply:
Cool, thanks